iree/compiler/Dialect/LinalgExt/IR/LinalgExtInterfaces.cpp

```diff
@@ -45,10 +40,11 @@ LogicalResult verifyLinalgExtOpInterface(Operation *op) {
       }
     }
   } else {
-    for (auto en : llvm::enumerate(linalgExtOp.inputs())) {
-      if (!en.value().getType().isa<MemRefType>()) {
+    for (auto en : llvm::enumerate(linalgExtOp.getInputOperands())) {
+      if (!linalgExtOp.isScalar(en.value()) &&
```

Comment:
Maybe also add a `hasBufferSemantics` to the `LinalgExtOp` interface. That will make this a bit cleaner.

Reply:
We do have the method, use the method instead.